lib/macroni/ASTConsumers/Kernel/KernelASTConsumer.cpp

```diff
@@ -46,13 +50,20 @@ void KernelASTConsumer::HandleTranslationUnit(clang::ASTContext &Ctx) {
   // Rewrite rcu_read_lock()/unlock() calls into critical sections.
   rewrite_rcu(&driver->mcontext(), mod);
 
-  // Print the result
+  // Print the result.
+
+  mlir::OpPrintingFlags flags;
+  // Only print original locations if the flag is enabled.
+  flags.enableDebugInfo(m_print_locations, false);
 
-  mod->print(llvm::outs());
+  mod->print(llvm::outs(), flags);
```

Comment:
Why is `ASTConsumer` also responsible for printing? Pull out, that's the job of its caller/user.